src/pcapi/core/bookings/repository.py

```diff
@@ -176,7 +176,7 @@ def find_expiring_bookings_ids() -> Query:
     return find_expiring_bookings().order_by(Booking.id).with_entities(Booking.id)
 
 
-def find_soon_to_be_expiring_booking_ordered_by_user(given_date: date = None) -> Query:
+def find_soon_to_be_expiring_non_eac_booking_ordered_by_user(given_date: date = None) -> Query:
```

Comment:
```suggestion
def find_soon_to_be_expiring_non_educational_booking_ordered_by_user(given_date: date = None) -> Query:
```
:warning: cela va être le cas partout
EAC n'est pas un terme existant dans notre code !!

Reply:
Voire d'ailleurs, plutôt que parler par le négatif, parler de individual_bookings